src/block/block_ckpt_scan.c

```diff
@@ -0,0 +1,345 @@
+/*-
+ * Copyright (c) 2014-2019 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+/*
+ * In historic WiredTiger files, it wasn't possible to open standalone files,
```

Comment:
The first `files` should be `databases`?

Reply:
Fixed in c55b7c2.